sdk/src/main/java/com/hcaptcha/sdk/HCaptchaConfig.java

```diff
@@ -100,4 +100,10 @@
      */
     @Builder.Default
     private String customTheme = null;
+
+    /**
+     * Reset hCaptcha on errors
+     */
+    @Builder.Default
+    private Boolean resetOnError = false;
```

Comment:
The issue states to reset on `timeout`. Shall the naming be updated to `resetOnTimeout`?

Reply:
@DSergiu good proposal, BTW maybe we should make it more like iOS and implement actual `resetOnError` not only for timeouts, @e271828- how do you think?

P.S. If not I'm totally agree to rename it

Reply:
As long as it's accurate and consistent between iOS/Android, I don't have a preference.

Reply:
I think `resetOnError` may be the same as on iOS and we will have consistent behavior on both platforms, @DSergiu how do you think?

Reply:
The 2 options behave differently. We can have both `resetOnError` and `resetOnTimeout` as separate configurable options.

Reply:
ok, let's stay with `resetOnTimeout` only to not introduce more differences between SDK